core/src/test/java/com/opensymphony/xwork2/ognl/SecurityMemberAccessTest.java

```diff
@@ -178,23 +178,6 @@ public void testMiddleOfInheritanceExclusion3() throws Exception {
         assertTrue("barLogic() from BarInterface isn't accessible!!!", accessible);
     }
 
-    @Test
-    public void testMiddleOfInheritanceExclusion4() throws Exception {
```

Comment:
Superinterfaces and superclasses of excluded classes are no longer also excluded. I'm not too sure of a real world example where this behaviour might actually be useful/sensible. Or where you wouldn't just add to the exclusion list the desired superinterfaces/superclasses instead?

If this behaviour is genuinely useful, it would be better if we moved such interfaces/classes to a separate exclusion list to keep the regular excluded class list performant.

Reply:
I cannot recall the exact use case, yet we can try to relax this behaviour hopping it was just my paranoid mind ;)